lib/enigma.rb

```diff
@@ -57,4 +57,20 @@ def generate_key
     end
     key_number
   end
+
+  def user_input
+    ARGV
+  end
+
+  def read_from_txt(filepath = user_input.first)
```

Comment:
The default value for the argument is what will be the most likely use cause for this as I planned it to not use an argument but decided on the default value to add flexibility. This also makes testing easier as an argument can be put in to read a file that is not the user input filepath.